openslides_backend/models/user.py

```diff
@@ -8,14 +8,76 @@ class User(Model):
     Model for users.
 
     There are the following reverse relation fields:
-        TODO
+    group_$<meeting_id>_ids: (group/user_ids)[];
```

Comment:
Please remove this file completly. Models are now in autogenerated models.py

Reply:
Done.